dbt/models/default/default.vw_pin_status.sql

```diff
@@ -0,0 +1,154 @@
+            OR SUBSTR(com.user16, 1, 2) = 'AI'
+            OR SUBSTR(com.user4, 1, 3) = 'SAP'
+            OR com.user13 IN ('49', '50', '51')
+            OR aprval.ecf IS NOT NULL
+            OR SUBSTR(dwel.user16, 1, 2) = 'AI'
+            OR admn.excode = 'INCV', FALSE
+        )) AS is_ahsap,
+        /* This column compares classes for parcels across dweldat, comdat, and
+        oby in order to construct a mixed use indicator. Note that we do not
+        consider EX or vacant land in oby.class to determine whether a parcel is
+        residential or commercial. */
+        MAX(
+            CASE
+                /* If a parcel exists in dweldat and comdat, or dweldat and oby
+                and class in oby is not vacant land, exempt, or class 2, then
+                it's mixed use. */
+                WHEN
+                    dwel.class IS NOT NULL
+                    AND (
+                        com.parid IS NOT NULL
+                        OR (
+                            oby.class IS NOT NULL
+                            AND (SUBSTR(oby.class, 1, 1) NOT IN ('1', '2'))
+                            AND oby.class NOT IN ('OA2', 'EX')
+                        )
+                    )
+                    THEN TRUE
+                    -- If a parcel is class 2 in oby and exists in comdat, then
+                    -- it's mixed use.
+                WHEN
+                    (SUBSTR(oby.class, 1, 1) = '2' OR oby.class = 'OA2')
+                    AND com.class IS NOT NULL
+                    THEN TRUE
+                ELSE FALSE
+            END
+        ) AS is_mixed_use
+    FROM {{ source('iasworld', 'pardat') }} AS par
+    LEFT JOIN {{ source('iasworld', 'comdat') }} AS com
+        ON par.parid = com.parid
+        AND par.taxyr = com.taxyr
+        AND com.cur = 'Y'
+        AND com.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'dweldat') }} AS dwel
+        ON par.parid = dwel.parid
+        AND par.taxyr = dwel.taxyr
+        AND dwel.cur = 'Y'
+        AND dwel.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'oby') }} AS oby
+        ON par.parid = oby.parid
+        AND par.taxyr = oby.taxyr
+        AND oby.cur = 'Y'
+        AND oby.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'land') }} AS land
+        ON par.parid = land.parid
+        AND par.taxyr = land.taxyr
+        AND land.cur = 'Y'
+        AND land.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'aprval') }} AS aprval
+        ON par.parid = aprval.parid
+        AND par.taxyr = aprval.taxyr
+        AND aprval.cur = 'Y'
+        AND aprval.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'exadmn') }} AS admn
+        ON par.parid = admn.parid
+        AND par.taxyr = admn.taxyr
+        AND admn.cur = 'Y'
+        AND admn.deactivat IS NULL
+    WHERE par.cur = 'Y'
+        AND par.deactivat IS NULL
+    GROUP BY par.parid, par.taxyr
+)
+
+SELECT
+    pdat.parid AS pin,
+    pdat.class,
+    pdat.taxyr AS year,
```

Comment:
**nitpick:** Let's swap the position of these two columns for better alignment with the other views.